o-fish-ios/Views/ReportFlow/Vessel/VesselView.swift

```diff
@@ -133,6 +133,7 @@ struct VesselView: View {
         self.vessel.homePort = self.initialVessel.homePort
         self.vessel.nationality = self.initialVessel.nationality
         self.showingPrefilledAlert = false
+        informationComplete = !vessel.isEmpty
```

Comment:
is this related to PR?

Reply:
This fixed bug with warning state, when you choose prefill vessel information (quick fix)